notebooks/Analyse_curated_playlist.ipynb

```diff
@@ -3,22 +3,20 @@
   {
```

Comment:
<p>wow that mean is amazing!</p>

 _Reply via <a href='https://app.reviewnb.com/tomsyouruncle/DJ_suggest/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>